search_by_car.py

```diff
@@ -0,0 +1,61 @@
+"""Take in a command line argument in the form:
+python search_by_car.py <db_filename>.sqlite <car_name>
```

Comment:
how would you refactor the api to this function to allow streaming in? i.e.

cat users.txt | python search_by_car.py

?

for instance, would you still take things in as a command line arg, or standard in?